diff --git a/node_modules/slash-create/lib/structures/interfaces/messageInteraction.d.ts b/node_modules/slash-create/lib/structures/interfaces/messageInteraction.d.ts
index df895c3..f2a5878 100644
--- a/node_modules/slash-create/lib/structures/interfaces/messageInteraction.d.ts
+++ b/node_modules/slash-create/lib/structures/interfaces/messageInteraction.d.ts
@@ -132,7 +132,7 @@ export interface EditMessageOptions {
 /** A file within {@link EditMessageOptions}. */
 export interface MessageFile {
     /** The attachment to send. */
-    file: Buffer;
+    file: Blob | Uint8Array | File;
     /** The name of the file. */
     name: string;
 }
